packaging/opensuse-42.2/snapd.spec

```diff
@@ -117,11 +127,22 @@ export CXXFLAGS
 %build
 # Build golang executables
 %goprep %{import_path}
+
+%if 0%{?with_test_keys}
+# The %gobuild macro doesn't allow us to pass any additional parameters
+# so we we have to invoke `go install` here manually.
+export GOPATH=%{_builddir}/go:%{_libdir}/go/contrib
+export GOBIN=%{_builddir}/go/bin
+go install -s -v -p 4 -x -tags withtestkeys github.com/snapcore/snapd/cmd/snapd
```

Comment:
Nitpick, can you use long forms or document the meaning of the switches above in a comment, for the next reader?